File: nos2jdbc-tutorial-spring/src/main/java/nos2jdbc/tutorial/spring/entity/nonauto/YmItemMember.java
package nos2jdbc.tutorial.spring.entity.nonauto;

import jakarta.persistence.Entity;
import jakarta.persistence.OneToOne;
import nos2jdbc.annotation.NoFk;
import nos2jdbc.annotation.NonAuto;

@Entity @NonAuto
public class YmItemMember {
    public Long memberId;
    public String memberName;
    @OneToOne @NoFk
    public YmItem ymItem;
}

File: nos2jdbc-tutorial-kotlin-spring/src/main/kotlin/nos2jdbc/tutorial/kotlinspring/entity/nonauto/YmItemMember.kt
package nos2jdbc.tutorial.kotlinspring.entity.nonauto

import jakarta.persistence.Entity
import jakarta.persistence.OneToOne

import nos2jdbc.annotation.NoFk
import nos2jdbc.annotation.NonAuto

@Entity @NonAuto
class YmItemMember {
    var memberId: Long? = null
    var memberName: String? = null
    @OneToOne @NoFk
    var ymItem: YmItem? = null
}
